crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -489,6 +489,16 @@ en:
         raid10: 'RAID 10'
       raid_disks_selected: 'Failed to save %{node}: Not enough disks selected for this RAID level'
 
+  upgrade:
+    upgrade:
+      nodes_not_ready: | 
```

Comment:
If you want the text to be here, remove the one from `installer` part, and adapt the usage (https://github.com/crowbar/crowbar-core/pull/490/files#diff-006ea8943f6241893e1af682a8f42073L329)